tests/test_misc.py

```diff
@@ -1727,7 +1727,7 @@ def mock_fail(*args):
     l1.daemon.rpcproxy.mock_rpc('getblockhash', mock_fail)
     l1.daemon.rpcproxy.mock_rpc('estimatesmartfee', mock_fail)
 
-    l1.daemon.start(wait_for_initialized=False)
+    l1.daemon.start(wait_for_initialized=False, stderr_redir=True)
```

Comment:
I don't think this should make a difference?

Unless these errors are going to stderr, in which case wait_for_logs is not the right thing 

This test flakes for me locally all the time too..